python/pyspark/sql/dataframe.py

```diff
@@ -4100,7 +4256,7 @@ def toDF(self, *cols: "ColumnOrName") -> "DataFrame":
         Parameters
         ----------
         cols : str
-            new column names
+            new column names. The length of the list needs to be the same as the number of columns in the initial :class:`DataFrame`
```

Comment:
Seems like it exceeds the 100 lines, which violates `flake8` rule.

```shell
starting flake8 test...
flake8 checks failed:
./python/pyspark/sql/dataframe.py:4250:101: E501 line too long (128 > 100 characters)
        """
        Returns a best-effort snapshot of the files that compose this :class:`DataFrame`.
        This method simply asks each constituent BaseRelation for its respective files and
        takes the union of all results. Depending on the source relations, this may not find
        all input files. Duplicates are removed.

        new column names. The length of the list needs to be the same as the number of columns in the initial :class:`DataFrame`

        .. versionadded:: 3.1.0

        Returns
        -------
        list
            List of file paths.

        Examples
        --------
        >>> df = spark.read.load("examples/src/main/resources/people.json", format="json")
        >>> len(df.inputFiles())
        1
        """

                                                                                        ^
1     E501 line too long (128 > 100 characters)
```

Reply:
You can run `dev/lint-python` to check if the static analysis is passed.

Reply:
done